src/Microsoft.NetCore.Analyzers/Core/Security/DoNotDisableRequestValidation.cs

```diff
@@ -76,9 +95,10 @@ public override void Initialize(AnalysisContext context)
                                 symbolAnalysisContext.ReportDiagnostic(
                                     symbol.CreateDiagnostic(
                                         Rule,
+                                        symbol is INamedTypeSymbol namedTypeSymbol ? namedTypeSymbol.TypeKind.ToString() : symbol.Kind.ToString(),
```

Comment:
> namedTypeSymbol.TypeKind.ToString() : symbol.Kind.ToString() [](start = 84, length = 61)

These enum value names aren't going to be localized.  Maybe just give the name, and don't mention in the message whether it's a method or class.